src/index.html

```diff
@@ -1,17 +1,212 @@
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            required
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="form">
+
+        <legend>Registration</legend>
+
+        <div class="form__item">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          ><br>
+        </div>
+
+        <div class="form__item">
+          <label for="password">Password</label>
+          <input
+              type="password"
+              name="password"
+              id="password"
+              minlength="7"
+              maxlength="20"
+              required
+          ><br>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="form">
+
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__item">
+
+          <label for="pet">Do you love cats?</label>
+
+          <input
+            type="radio"
+            name="pet" id="yes"
+            value="Yes"
+          >
+          <label for="yes">Yes</label>
+
+          <input
+            type="radio"
+            name="pet"
+            id="no"
+            value="No"
+          >
+          <label for="no">No</label><br>
+
+        </div>
+
+        <div class="form__item">
+          <label for="favColor">What is your favorite color?</label>
+          <input
+            type="color"
+            name="favColor"
+            id="favColor"
+          ><br>
+        </div>
+
+        <div class="form__item">
+          <label for="sleep">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="sleep"
+            id="sleep"
+            step="2"
+          ><br>
+        </div>
+
+        <div class="form__item">
+          <label for="favCar">What are your favorite brand of cars?</label>
+          <select
+            name="favCar"
+            id="favCar"
+            multiple>
+            <option value="favCar">BMW</option>
+            <option value="favCar">Audi</option>
+            <option value="favCar">Lada</option>
+          </select><br>
+        </div>
+
+        <div class="form__item">
+          <label for="workRate">How do you rate our work?</label>
+          <input
+            type="range"
+            name="workRate"
+            id="workRate"
+            min="1"
+            max="5"
+            step="1"
+            value="1"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="form">
+
+        <legend>Additional info:</legend>
+
+        <div class="form__item">
+
+          <label for="comment">Comments:</label>
+          <textarea
+          name="comment"
+            id="comment"
+            cols="15"
+            rows="2">
+          </textarea><br>
+
+        </div>
```

Comment:
Don't add line breaks after last child element
```suggestion
          </textarea><br>
        </div>
```

Reply:
Fix that all over the code

Reply:
Fixed